lib/pangea/extensions/pangea_room_extension/room_information_extension.dart

```diff
@@ -1,6 +1,17 @@
 part of "pangea_room_extension.dart";
 
 extension RoomInformationRoomExtension on Room {
+  Future<int> get _numNonAdmins async {
```

Comment:
Sneaky bug I found in testing - because leaveIfFull is called after the user has joined the room, they are a member, so they're counted in this number. So, if they are the last member joining before capacity is reached, it will look like the room is already full.